hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterHeartbeatManager.java

```diff
@@ -100,7 +101,7 @@
     private final int icmpIntervalMillis;
     private final int icmpMaxAttempts;
 
-    @Probe(name = "lastHeartbeat")
+    @Probe(name = CLUSTER_METRIC_HEARTBEAT_MANAGER_LAST_HEARTBEAT)
```

Comment:
Does this probe stand for time? If yes, how about `MS` unit here?

Reply:
Yep, added `MS`.